apps/omg_performance/test/omg_performance/byzantine_events/mutations_test.exs

```diff
@@ -0,0 +1,45 @@
+# Copyright 2019 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.Performance.ByzantineEvents.MutationsTest do
+  use ExUnit.Case, async: true
+
+  alias OMG.Performance.ByzantineEvents.Mutations
+  alias OMG.State.Transaction
+
+  @eth OMG.Eth.RootChain.eth_pseudo_address()
+
+  setup do
+    alice = OMG.TestHelper.generate_entity()
+    {:ok, %{alice: alice}}
+  end
+
+  describe "mutate_txs/2" do
+    test "should produce a different signed transaction every time", %{alice: alice} do
+      tx = OMG.TestHelper.create_encoded([{1000, 0, 0, alice}], @eth, [{alice, 1}])
```

Comment:
gaahh I hate these helper functions. they're really good at hiding what they're doing. bad naming doesn't help either!

Reply:
would `OMG.TransactionHelper.create_encoded` help? Mind that their intention is to provide a short-hand way to create transactions needed to test various things, so the syntax is concise by intent.

anyway, I'd suggest taking this to a separate PR.